c++/src/kj/async.h

```diff
@@ -1169,6 +1169,13 @@ class WaitScope {
   //
   // Not supported in fibers.
 
+  uint poll(uint maxTurnCount);
```

Comment:
Sure, I am happy with that; the original change was an overload so as to not impact existing code. Will amend commit and push.

Reply:
I must have been tired and forgot to run tests against the latest iteration at the time when I raised the PR. There was a bug where it was counting down from zero rather than up. Fixed the bug and changed to default valued function prototype.